examples/all-clusters-app/nxp/mw320/BUILD.gn

```diff
@@ -91,6 +91,7 @@ mw320_executable("shell_mw320") {
     "${chip_root}/examples/energy-management-app/energy-management-common/src/ElectricalPowerMeasurementDelegate.cpp",
     "${chip_root}/examples/energy-management-app/energy-management-common/src/EnergyEvseDelegateImpl.cpp",
     "${chip_root}/examples/energy-management-app/energy-management-common/src/EnergyEvseManager.cpp",
+    "${chip_root}/examples/energy-management-app/energy-management-common/src/EnergyEvseTargetsStore.cpp",
```

Comment:
For the gn clusters, can this not just be added to the data model gni file that pulls in the server files?

Reply:
I didn't put them in there because these sit as 'examples' in the 'delegate' space whereas things in src/app/cluster/server area are already in the data_model.gni file (for the lower level cluster handling of attributes and commands).

Reply:
Marking this as resolved now.